### DISCUSSION TITLE
How to disable rate limiting settings

### DISCUSSION THREAD

**Discussion by zhaoshouli** [jlesage/docker-firefox#265] [Q&A] [answered]
After frequent access, Xvnc temporarily blacklists the accessing IP. Can this be disabled through a parameter?
Other answers, thank you
![Uploading image.png…]()

**Comment by jlesage** [comment]
This is currently no such parameter. What is the use case for this ?

**Reply by zhaoshouli** [reply]
![novnc](https://github.com/user-attachments/assets/429b971c-e9a3-4ad5-baff-fba97dc2fac2)
Refreshing the interface multiple times results in access failure, and the container logs show this message.

**Reply by jlesage** [reply]
Can you share the full log ? No matter how frequently I refresh the web page, I don't see the problem. Do you have the secure connection enabled ?

**Reply by zhaoshouli** [reply]
Sorry, I didn't provide a clear explanation. The issue occurs when I access the image port 5800 through my laptop's browser, and it happens when I frequently refresh the access in the browser, not when refreshing the service inside the container.

**Reply by jlesage** [reply, marked as answer]
You were clear, this is what I understood :) An IP should be blacklisted only when there are multiple connection failures... not by re-connections (triggered by the refreshes).

**Reply by zhaoshouli** [reply]
Understood. Thank you very much for your explanation.
